### PR TITLE
Add ability to exclude folders and files to check

### PR DESCRIPTION
**Pull request by adnikiforov**
From the issue https://github.com/reviewdog/action-hadolint/issues/16
How it works: we exclude files and folders using consecutive `--exclude` rules for `git ls-files`. Like so:

```
with:
  exclude: |
    /vendor/*
    Dockerfile.*
```

That's how it looks like:

```
git ls-files --exclude='Dockerfile*' --ignored 
Dockerfile
vendor/Dockerfile
vendor/Dockerfile.bak

git ls-files --exclude='Dockerfile*' --ignored --exclude='!/vendor/Dockerfile*'
Dockerfile

git ls-files --exclude='Dockerfile*' --ignored --exclude='!/vendor/*'           
Dockerfile

git ls-files --exclude='Dockerfile*' --ignored --exclude='!/vendor/Dockerfile.*'
Dockerfile
vendor/Dockerfile
```

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -3,7 +3,13 @@
 cd "$GITHUB_WORKSPACE"
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
-git ls-files --exclude='Dockerfile*' --ignored | xargs hadolint \
+EXCLUDES=""
+for exclude_path in $INPUT_EXCLUDE; do
+  EXCLUDES="EXCLUDES --exclude='!$exclude_path'"
```
Comment (mgrachev) [inline entrypoint.sh:8]:
```suggestion
  EXCLUDES="$EXCLUDES --exclude='!$exclude_path'"
```
Is the dollar sign optional? Will this work without it?

**Review by mgrachev — COMMENTED** [review]
@adnikiforov I like that implementation ❤️

I have left some comments 👀

File: entrypoint.sh (hunk 1 above)
Comment (adnikiforov) [reply entrypoint.sh:8]: No, that's right, I missed this one.

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@adnikiforov Great job! 🚀 Thank you! ❤️

**Pull request merged**